go.mod

```diff
@@ -46,6 +46,6 @@ require (
 	github.com/shopspring/decimal v1.3.1 // indirect
 	github.com/spf13/cast v1.4.1 // indirect
 	golang.org/x/crypto v0.0.0-20220411220226-7b82a4e95df4 // indirect
-	golang.org/x/sys v0.0.0-20211216021012-1d35b9e2eb4e // indirect
```

Comment:
I didn't change this. :raised_eyebrow:

Reply:
Second group, it's an indirect dependency, updated in `go-jsonnet`